src/Authentication/Authentication.Core/Microsoft.Graph.Authentication.Core.csproj

```diff
@@ -12,14 +13,21 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.25.0" />
-    <PackageReference Include="Azure.Identity" Version="1.6.1" />
-    <PackageReference Include="Microsoft.Graph.Core" Version="2.0.11" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.51.0" />
-    <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.27.0" />
+    <PackageReference Include="Azure.Identity" Version="1.7.0" />
+    <PackageReference Include="Azure.Identity.BrokeredAuthentication" Version="1.0.0-beta.3" />
+    <PackageReference Include="Microsoft.Graph.Core" Version="2.0.14" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.47.2" />
+    <PackageReference Include="Microsoft.Identity.Client.Broker" Version="4.47.2-preview" />
+    <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.24.0" />
+    <PackageReference Include="Microsoft.Identity.Client.NativeInterop" Version="0.13.0" />
+    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="6.27.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Logging" Version="6.27.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.27.0" />
```

Comment:
We no longer use the library due to https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/1818#issuecomment-1444396479. Please remove the extra libraries that are being added by this PR. I believe the PR only needs `Azure.Identity.BrokeredAuthentication`, `Microsoft.Identity.Client.Broker`, and `Microsoft.Identity.Client.NativeInterop` to run.